docs/learn/documentation/versioned/jobs/configuration-table.html

```diff
@@ -156,13 +156,20 @@
                         others' checkpoints, and perhaps interfere with each other in other ways.
                     </td>
                 </tr>
-                <tr>
                     <td class="property" id="job-coordinator-system">job.coordinator.system</td>
                     <td class="default"></td>
                     <td class="description">
                         <strong>Required:</strong> The <span class="system">system-name</span> to use for creating and maintaining the <a href="../container/coordinator-stream.html">Coordinator Stream</a>.
                     </td>
                 </tr>
+                <tr>
+                    <td class="property" id="job-default-system">job.default.system</td>
```

Comment:
@jmakes as for the scope of configuration, I thought that you were creating one more level above job when you were working on the pipeline feature. Anyways, w/ the new API that potentially create a multi-stage Samza application (as in SAMZA-1041), I believe that we need to have the following scopes of configuration variables now: application ==> job ==> task, w/ the definition that application configuration include all common configures for the multi-stage Samza application that may have multiple jobs, job configuration include all common configuration for Samza containers in a job, and task configuration include configuration at task level. If that sounds good to you, let me open a new JIRA regarding to creating the new scope in Samza configuration.